.github/workflows/analyzer.yml

```diff
@@ -17,6 +17,9 @@ on:
       - analyzer/**
       - shared/**
 
+  schedule:
+    - cron: 0 1 * * 1
```

Comment:
Because we sometimes don't touch some parts of the code for months and we get some surprises from time to time.

Such surprises happen because we don't pin the dev tools, which is fine by me. We also make sure the environment didn't break.